test/socket.test.js

```diff
@@ -18,7 +18,10 @@ describe('socket', () => {
             socket = null;
         }
 
-        server.close(done);
+        server.close(err => {
+            // delay for freeing up port
```

Comment:
Erp, yea, sorry, reading during meetings. Kinda unfortunate that (presumably?) the callback fires before the listener is actually closed.